packages/confluent-kafka-receive/README.md

```diff
@@ -44,10 +44,9 @@ class KafkaExample {
   }
 }
 
-KafkaExample.registeredConsumerWorkflow = DBOS.registerWorkflow(
-  KafkaExample.registeredConsumerWorkflow,
-  'registeredConsumerWorkflow',
-);
+KafkaExample.registeredConsumerWorkflow = DBOS.registerWorkflow(KafkaExample.registeredConsumerWorkflow, {
```

Comment:
Removing the optional name parameter from tests is optional, but we should absolutely remove the explicit name parameter from readmes.